setup.py

```diff
@@ -10,6 +10,7 @@
     url="https://github.com/sputt/req-compile",
```

Comment:
We'll need to bump the version in this PR, I think this can be released standalone

Reply:
Done and confirmed as well that py.typed ends up in the wheel
```
  inflating: req_compile/py.typed    
```